src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java

```diff
@@ -5,6 +5,7 @@
 
 import javax.annotation.Nonnull;
 
+import io.github.thebusybiscuit.slimefun4.implementation.items.backpacks.*;
```

Comment:
No star imports. And please follow the import format.

Reply:
Intellij changed it

Reply:
Still: Don't use star imports.